config/images/images.yaml

```
@@ -66,6 +66,21 @@ images:
   tags:
   - v1.24.1
   - v1.26.4
+# credativ/vali
+- source: ghcr.io/credativ/vali
+  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
+  tags:
+  - v2.2.13
+# credativ/valitail
+- source: ghcr.io/credativ/valitail
+  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
+  tags:
+  - v2.2.13
+# credativ/plutono
+- source: ghcr.io/credativ/plutono
+  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
+  tags:
+  - v7.5.28
```

These are all images related to gardener/gardener's images.yaml file (https://github.com/gardener/ci-infra/blob/092dc4b86602d0e05988eb74839995f5d5c94caa/config/images/images.yaml#L2-L5). Hence no need of the comments.

```suggestion
- source: ghcr.io/credativ/vali
  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
  tags:
  - v2.2.13
- source: ghcr.io/credativ/valitail
  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
  tags:
  - v2.2.13
- source: ghcr.io/credativ/plutono
  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
  tags:
  - v7.5.28
```